<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GEPinger 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ipAddr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ping">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>IP Header</string>
    </property>
    <widget class="QLineEdit" name="ipTos">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TOS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TTL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipTtl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Don't fragment</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>ICMP Header</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sid">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SN start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="count">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="timeout">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>inter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inter">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="datasize">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Datasize</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="Output">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>391</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ping</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ipAddr</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_click()</slot>
 </slots>
</ui>
